cms-adapter/src/Rsbc.Dmf.CaseManagement.Service/AutoMapper.cs

```diff
@@ -47,8 +47,9 @@ public MappingProfile()
             CreateMap<Dto.Case, Case>();
             CreateMap<Dto.Person, Person>();
             CreateMap<Dto.DocumentType, DocumentType>();
-            CreateMap<Document, DmerCase>()
+            CreateMap<Dto.Document, DmerCase>()
                 .ForMember(dest => dest.Status, opt => opt.MapFrom(src => src.DmerStatus))
+                .ForMember(dest => dest.DocumentId, opt => opt.MapFrom(src => src.DocumentId))
```

Comment:
next time let's just call it "Document.Id" instead of "Document.DocumentId". It's redundant, like Hungarian notation. But for now not really a concern.